### DISCUSSION TITLE
Fields have to be refilled/reselcted if there is an error...

### DISCUSSION THREAD

**Discussion by andrewdwallo** [filamentphp/filament#4219] [Help]
While filling in a form, if there is an error, I have to reselect and fill every field again even though the fields show that they are filled in... This usually only happens if it is my first time filling in the form, but this will cause issues for customers I plan on having in the future., and it would definitely help you. Just needed some advice on how to fix this...

**Comment by danharrin** [comment]
Validation errors should not do this... only 500s. Its Livewire behaviour, not Filament

**Reply by andrewdwallo** [reply]
Would you happen to know how to fix it considering a large amount of filament uses livewire?

**Reply by danharrin** [reply]
The fix is to make sure your app doesnt throw 500s? I dont know why you would want your users to trigger those
